packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -27,6 +28,18 @@ const normalizeContentValue = (value: string) => {
   return value;
 };
 
+/**
+ * Picks identifier (`mixin`, `mixin1`, `mixin2`) from call expression as given in below example
+ * `css={{ ':hover': mixin(), border: sizes.mixin1(), ...mixin2(), ...mixin3 }}` or `css={\`\${mixin()}\`}`
+ * If it's not a call expression, it will pick the identifier (`mixin3` in above example)
+ *
+ * @param node Any Expression or Identifier
+ */
+const getIdentifierFromCallExpression = (node: t.Expression) =>
+  // TODO: Handle arguments when we pick mixin arguments story. Right now
+  // we are only picking function without any arguments.
+  t.isCallExpression(node) && node.arguments.length === 0 ? node.callee : node;
```

Comment:
what is `node` result? since this is returning `node` as the fallback let's come up with a name that represents what the function is doing - any ideas? (since it doesnt always return an identifier)